app/models/bigbluebutton_server_config.rb

```diff
@@ -0,0 +1,33 @@
+require 'bigbluebutton_api'
+
+class BigbluebuttonServerConfig < ActiveRecord::Base
+  include ActiveModel::ForbiddenAttributesProtection
+
+  belongs_to :server, class_name: 'BigbluebuttonServer'
+  validates :server_id, presence: true
+
+  def get_available_layouts
+    if self.available_layouts.blank?
+      # Locally we store it as a comma-separated string.
```

Comment:
There's a `serialize` option in Rails that I believe would be better for the `available_layouts`. Then you could just set an array on it and rails will serialize/deserialize whenever needed. So manual joins and splits are not needed.
We already use it in some places, see https://github.com/mconf/mconf-web/blob/master/app/models/site.rb#L10